chapters/annotations.tex

```diff
@@ -411,8 +427,8 @@ \section{Symbolic Processing}\label{annotations-for-symbolic-processing}\label{s
 \end{center}
 
 \begin{annotationdefinition}[Evaluate]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"Evaluate" "=" ( false | true )
+\begin{synopsis}\begin{lstlisting}
+/*literal*/ constant Boolean Evaluate;
```

Comment:
It is difficult to express the default for this with `Ternary`.

Reply:
As a value, yes. But I added the text that the tools decide in that case - which correspond to the current default behavior.